envoy/http/filter.h

```diff
@@ -448,19 +448,20 @@ class StreamDecoderFilterCallbacks : public virtual StreamFilterCallbacks {
   virtual MetadataMapVector& addDecodedMetadata() PURE;
 
   /**
-   * Called with 100-Continue headers to be encoded.
+   * Called with 1xx headers to be encoded.
```

Comment:
Based on the impl it seems we only pass a specific set of "special" 1xx headers here, should this be included in the comments?

Reply:
yeah I was trying to be lazy and not have to update all the comments in the next PR.  I'll stop being lazy :-P